### PR TITLE
Calculate sum of all digits from the string and find the cheapest book

### PR DESCRIPTION
**Pull request by volodymyrshkliar**

### CODE REVIEW

File: volodymyr_shkliar/find_the_cheapest_book.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        'name': 'Fluent Python',
+        'price': 650
+    }
+]
+
+
+def min_price(data: list) -> str:
+    data.sort(key=lambda x: x['price'])
+
+    return "The cheapest book is '{}'. It costs {} grn.".format(data[0].get('name'), data[0].get('price'))
```
Comment (pivancy) [inline volodymyr_shkliar/find_the_cheapest_book.py]: Well, to make your code much better I suggest setting the cheapest book into a separate variable.

Your string formatting should look better after this change.

**Comment by pivancy** [comment]
This name of pull request is not self-descriptive enough.

It should be something like:

`Find the cheapest book and calculate the sum of digits from the number`, etc.

**Comment by pivancy** [comment]
Also, you didn't add me as an `assignee` as it's described in the README file of this repository.

There should be a lot of reviewers, but somebody should be responsible for the final decision(assignee).

File: volodymyr_shkliar/find_the_cheapest_book.py (hunk 1 above)
Comment (volodymyrshkliar) [reply volodymyr_shkliar/find_the_cheapest_book.py]: Is it good to write name, price = data[0].values() ?

Comment (pivancy) [reply volodymyr_shkliar/find_the_cheapest_book.py]: Nope, it's a bad idea.

Comment (volodymyrshkliar) [reply volodymyr_shkliar/find_the_cheapest_book.py]: Done.

**Comment by pivancy** [comment]
👍

**Pull request merged**